File: SafeDalat_API/SafeDalat_API/Model/DTO/QA/AnswerResponseDTO.cs
namespace SafeDalat_API.Model.DTO.QA
{
    public class AnswerResponseDTO
    {
        public int AnswerId { get; set; }
        public string Content { get; set; } = null!;
        public DateTime CreatedAt { get; set; }

        public int ResponderId { get; set; }
        public string ResponderName { get; set; } = null!;


        public string? DepartmentName { get; set; }
    }
}

File: DalatS/app/src/main/java/com/example/dalats/model/AnswerResponseDTO.java
package com.example.dalats.model;
import com.google.gson.annotations.SerializedName;
import java.io.Serializable;

public class AnswerResponseDTO implements Serializable {
    @SerializedName("answerId")
    private int answerId;
    @SerializedName("content")
    private String content;
    @SerializedName("responderName")
    private String responderName;
    @SerializedName("departmentName")
    private String departmentName;
    @SerializedName("createdAt")
    private String createdAt;

    public String getContent() { return content; }
    public String getResponderName() { return responderName; }
    public String getDepartmentName() { return departmentName; }
    public String getCreatedAt() { return createdAt; }
}